platform/darwin/src/http_file_source.mm

```diff
@@ -208,10 +208,19 @@ void cancel() {
         if (impl->accountType == 0 &&
             ([url.host isEqualToString:@"mapbox.com"] || [url.host hasSuffix:@".mapbox.com"])) {
             NSURLComponents *components = [NSURLComponents componentsWithURL:url resolvingAgainstBaseURL:NO];
-            NSArray *newQueryItems = @[
-                [NSURLQueryItem queryItemWithName:@"events" value:@"true"],
-                [NSURLQueryItem queryItemWithName:@"sku" value:MGLAccountManager.skuToken]
-            ];
+            
+            NSMutableArray *newQueryItems = [NSMutableArray arrayWithArray:@[
+                [NSURLQueryItem queryItemWithName:@"events" value:@"true"]
+                ]];
+
+            // Only add the token if we have enabled the accounts SDK
+            if (MGLAccountManager.isAccountsSDKEnabled)
+            {
+                NSString *skuToken = MGLAccountManager.skuToken;
+                if (skuToken) {
```

Comment:
Checking for `nil` here feels redundant to me, given that `MGLAccountManager.skuToken` shouldn't return `nil` if `MGLAccountManager.isAccountsSDKEnabled = YES`.